source/Templates/NewStep/Sashimi.NamingIsHard/source/Calamari.Tests/MyCommandFixtures.cs

```diff
@@ -0,0 +1,29 @@
+using System.IO;
+using System.Threading.Tasks;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.NamingIsHard.Commands;
+using Calamari.Tests.Shared;
+using NUnit.Framework;
+
+namespace Calamari.NamingIsHard.Tests
+{
+    [TestFixture]
+    public class MyCommandFixtures
```

Comment:
Add comments to explain when you'd write one of these tests over a Sashimi one

Reply:
Done